extra/callback_server.py

```diff
@@ -0,0 +1,30 @@
+from http.server import HTTPServer, BaseHTTPRequestHandler
+import sys
+
+
+class RequestHandler(BaseHTTPRequestHandler):
+    def do_POST(self):
+        content_length = int(self.headers['Content-Length'])
+        post_data = self.rfile.read(content_length)
+
+        print("Received POST request:")
+        print(post_data.decode('utf-8'))
+
+        self.send_response(200)
+        self.send_header('Content-type', 'text/plain')
+        self.end_headers()
+        self.wfile.write(b"{\"message\": \"Message Received\"}")
+
+
+def run_server(port=9888):
```

Comment:
The port needs to be a parameter exposed as an env variable, also if you have 9888 written out twice in this file, better to just put it as a constant at the top. Look at this example from `cache.py` as a "canonical" way to set this type of env var: `DEFAULT_TTL = int(get_environment_setting("CACHE_DEFAULT_TTL") or 24*60*60)`

Reply:
makes sense, just added the environment variable with a default value of 9888 as suggested
I'd like to keep the cmd argument option open, which means we first look at env var, default to 9888 if not set, and can change that either through setting the env var or passing it as an arg to the server when running it in the command line.